tests/upgrade/post/Makefile

```diff
@@ -0,0 +1,30 @@
+include ../../../common.mk
+
+# Check if Go's linkers flags are set in common.mk and add them as extra flags.
+ifneq ($(GOLDFLAGS),)
+	GO_EXTRA_FLAGS += -ldflags $(GOLDFLAGS)
+endif
+
+# Build code with jemalloc tag unless explicitly disabled (used by badgerdb).
+ifneq ($(OASIS_BADGER_NO_JEMALLOC), 1)
+	GO_EXTRA_FLAGS += -tags jemalloc
+endif
+
+# Set all target as the default target.
+all: build
+
+# Build.
+build:
+	@$(ECHO) "$(MAGENTA)*** Building oasis-test-runner...$(OFF)"
+	@$(GO) build $(GOFLAGS) $(GO_EXTRA_FLAGS) -o oasis-test-runner
```

Comment:
Should this be named something different to avoid confusion with the "regular" test runner? Maybe something like `oasis-test-upgrade`?